library/catalog/templates/books.html

```diff
@@ -12,7 +12,7 @@ <h1>Library page</h1>
                 <div class="zoom">
                     <a href={% url 'book-details' book.id %}>
                         <img style="padding: 10px 10px 10px 10px;max-height: 270px; width: 90%" src={{ book.link_img }}>
-                        <p style="margin-left: 10px">{{ book.title }}</p>
+                        <p style="margin-left: 10px; color: #323232">{{ book.title }}</p>
```

Comment:
Can we move CSS to the main CSS file ?

Reply:
I will change the books.html template in the next pull request, there is a lot of CSS on it